core/posts.go

```diff
@@ -21,19 +21,32 @@ import (
 
 // Constants for validation
 const (
-	PostTitleMaxCharacters    = 280
+	// PostStatusMaxCharacters - Maximum length of the status field of a post
+	PostStatusMaxCharacters = 280
+	// PostLongFormMaxCharacters - Maximum length of the longForm field of a post
 	PostLongFormMaxCharacters = 50000
-	MaxPostTags               = 50
-	PostTagsMaxCharacters     = 80
+	// MaxPostTags - Maximum number of tags a post can have
+	MaxPostTags = 50
+	// MaxPostChannels - Maximum number of channels a post can be addressed to
+	MaxPostChannels = 30
```

Comment:
> Are the channels here intending to be the same channels proposed in https://github.com/OpenBazaar/obips/blob/master/obip-0003.mediawiki?

No, these channels are akin to IRC channels. I opted to go with the 'channel' terminology in this refactor since:

1. Curation channels hadn't been implemented yet
2. Many in the team expressed interest in using another term for 'curation channels'

Reply:
> Is there any rationale behind 30 and not something larger?

Nothing particularly strong. I didn't want it too low that it would require a user making multiple posts to target all the channels they wanted, or so high that someone could spam dozens and dozens of channels at once. This value can be adjusted based on user behaviour.